static/pages/home/home.html

```diff
@@ -100,7 +100,7 @@ <h3 style="margin-top: 0;">Impacting students around the world</h3>
     <md-card-content layout="row" layout-align="center">
       <md-content layout="column" class="oppia-foundation-white-text oppia-foundation-green-background home-page-history-text-box">
         <p class="oppia-foundation-center-align-text">
-          Since launching Oppia.org in 2015, over 800,000 learners in more than 20 countries have used our platform to explore, grow, and learn.
+          Since launching Oppia.org in 2015, over one million learners in more than 20 countries have used our platform to explore, grow, and learn.
```

Comment:
Hmmm, I think I used the numeric count (`1,000,000`) over in Partnerships page. Do you want to apply this changes to `partnerships.html` too?

Reply:
Sure thing, easy do.

Reply:
Oh, do you want them both to say "one million" or "1,000,000"?

Reply:
I don't really have a strong preference here. I do feel like the numeric version has stronger impact. That is just me though.